packages/NodeTypeResolver/PHPStan/Scope/PHPStanNodeScopeResolver.php

```diff
@@ -319,6 +320,24 @@ private function processTernary(Ternary $ternary, MutatingScope $mutatingScope):
         $ternary->else->setAttribute(AttributeKey::SCOPE, $mutatingScope);
     }
 
+    /**
+     * @param Stmt[] $stmts
+     * @param callable(Node $node, MutatingScope $scope): void $nodeCallback
+     */
+    private function processNodesFromNodeScopeResolver(
+        array $stmts,
+        MutatingScope $mutatingScope,
+        callable $nodeCallback
+    ): void {
+        try {
+            $this->nodeScopeResolver->processNodes($stmts, $mutatingScope, $nodeCallback);
+        } catch (InvalidIdentifierName $invalidIdentifierName) {
+            if ($invalidIdentifierName->getMessage() !== 'Invalid identifier name "0"') {
```

Comment:
I think this should be fixed propperly. What is the root of this issue?

Reply:
I reverted the tweak https://github.com/rectorphp/rector-src/pull/2903/commits/14cbabfcff7c3ce4cb7f5d8f53d774b57d73c96a 

Now to make CI green,  https://github.com/rectorphp/rector-downgrade-php/pull/24 need to be merged first.

Reply:
:partying_face: